scripts/main.py

```diff
@@ -19,8 +30,19 @@ def main(args=None):
 
     if ns.infile[0] is not None:
```

Comment:
if statement should not be required if you add the `required` option in the argparse...

Reply:
turns out that as a positional argument (because I used `infile` as opposed to `--infile`), it's already required! So I don't need and cannot even use required=True. So I'm dropping the `if ns.infile[0] is not None:` check but leaving the parser as-is